README.md

```diff
@@ -5,7 +5,8 @@ Boilerplate to help you get started quickly with the Java client library for Act
 ### Setup Instructions
 
 #### Action Configuration
-1. From the [Actions on Google Console](https://console.actions.google.com/), add a new project (this will become your *Project ID*) > **Create Project** > Scroll down to the **More Options** section, and click on the **Conversational card**.
+1. From the [Actions on Google Console](https://console.actions.google.com/), add a new project (this will become your *Project ID*) > **Create Project**.
+1. Scroll down to the **More Options** section, and click on the **Conversational card**.
```

Comment:
Only "**Conversational**" needs to be in bold, rather than "**Conversational card**"